toolchain/symbiflow-toolchain/meta.yaml

```diff
@@ -0,0 +1,30 @@
+{% set version = '%s_%04i_%s'|format(GIT_DESCRIBE_TAG|replace('v','') or '0.X', GIT_DESCRIBE_NUMBER|int, GIT_DESCRIBE_HASH or 'gUNKNOWN') %}
+
+package:
+  name: symbiflow-toolchain
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/verilog-to-routing/vtr-verilog-to-routing.git
```

Comment:
This should be `https://github.com/SymbiFlow/symbiflow-arch-defs.git`

Reply:
Fixed